apiserver/plane/db/models/__init__.py

```
@@ -93,7 +93,7 @@
 
 from .estimate import Estimate, EstimatePoint
 
-from .inbox import Inbox, InboxIssue
+from .intake import Intake, IntakeIssue
```

Ruff

Redefinition of unused `Intake` from line 18

Remove definition: `Intake`

(F811)